### DISCUSSION TITLE
Weird input

### DISCUSSION THREAD

**Discussion by AverageTf2Enjoyer** [ppy/osu#20670] [Q&A]
i play good a song and the game force to miss arround 20 to 40 beats, and the accuracy hit so much 200 and 100 (it is not offset problem) its a lazer problem or need enable or dissable anything in the configuration ? (osu!mania)

**Comment by WindowsMeosu** [comment]
can you please specify? Is it that inputs aren't registering correctly? any chance you can send something like a video or a replay when this occurs? I'd like to think this is the cause of ghost-tapping or polling rate but the way you formatted it, it just appears that your keyboard or the game isn't registering inputs correctly.

**Reply by AverageTf2Enjoyer** [reply]
https://www.youtube.com/watch?v=tD0mz0EVrL0

**Reply by AverageTf2Enjoyer** [reply]
this is the problem that happens to me
